### PR TITLE
mediafield-attached media -use user friendly file name if exists

### PR DESCRIPTION
**Pull request by giannik**
When selecting a file using the attached media field it shows the technical name, which is not user friendly. with this change we use the uploaded file name if it exists.

before : 
image

after : 
image

### CODE REVIEW

File: src/OrchardCore.Modules/OrchardCore.Media/Views/MediaField-Attached.Edit.cshtml [hunk 1]
```diff
@@ -40,7 +40,7 @@
                 <a href="javascript:;" v-on:click="removeSelected" class="btn btn-secondary btn-sm" v-bind:class="{ disabled : !selectedMedia }"><i class="fa-solid fa-trash-alt" aria-hidden="true"></i></a>
             </div>
             <label class="selected-media-name form-label">
-                <code class="text-end" v-if="selectedMedia">@T["{{{{ selectedMedia.name }}}}{{{{ selectedMedia.mediaText === '' ? '' : ', ' + selectedMedia.mediaText }}}} ({{{{ isNaN(fileSize)? 0 : fileSize }}}} KB)"]</code>
+                <code class="text-end" v-if="selectedMedia">@T["{{{{ selectedMedia.attachedFileName !== null && selectedMedia.attachedFileName !== '' ? selectedMedia.attachedFileName : selectedMedia.name }}}} ({{{{ isNaN(fileSize)? 0 : fileSize }}}} KB)"]</code>
```
Comment (Piedone) [inline src/OrchardCore.Modules/OrchardCore.Media/Views/MediaField-Attached.Edit.cshtml:43]: Why did you remove `selectedMedia.mediaText`?

**Review by Piedone — CHANGES_REQUESTED** [review]
I sincerely apologize for us taking so much time here.

File: src/OrchardCore.Modules/OrchardCore.Media/Views/MediaField-Attached.Edit.cshtml (hunk 1 above)
Comment (giannik) [reply src/OrchardCore.Modules/OrchardCore.Media/Views/MediaField-Attached.Edit.cshtml]: my miss. added it
